subprojects/packagefiles/openblas/meson.build

```diff
@@ -0,0 +1,648 @@
+# Conventions:
+# _ implies that the variables are not meant to be used outside here
+# Optionals are applied from the top-level meson_options.txt
+# They are declared at the top
+# Typically derived from (in order) the CMakeLists.txt and Makefiles
+#
+# Installation:
+# meson setup build --buildtype release
+# meson compile -C build
+# meson install --prefix=$HOME/.local/lapack
+#
+# NOTE: This is still a work in progress, the Makefiles are canonical
+project('OpenBLAS', ['c', 'fortran'],
+        default_options: ['c_std=c99', 'pkgconfig.relocatable=true'],
```

Comment:
The relocatable option should not be set. The fact that it errors might be a bug, but subproject wraps should not set these options, instead they should be defined by the consuming project.